Sources/Public/Animation/LottieView.swift

```diff
@@ -380,4 +400,16 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     }
   }
 
+  private func clearAnimationIfNecessary() {
+    if loadAnimation != nil, showPlaceholderWhileReloading {
+      animationSource = nil
+    }
+  }
```

Comment:
I'm not sure about this specific method since it's tightly coupled to the `reloadAnimationTrigger` use case and it doesn't seem like we'd call it again for something else.

Maybe instead organization like this would make sense:

```suggestion
  private func reloadAnimationTriggerDidChange() {
    guard loadAnimation != nil else { return }
    
    if showPlaceholderWhileReloading {
      animationSource = nil
    }
    
    loadAnimationIfNecessary()
}
```

this feels more self-documenting to me, compared to `clearAnimationIfNecessary`